graylog2-web-interface/src/components/pipelines/ProcessingTimelineComponent.tsx

```diff
@@ -99,7 +137,7 @@ const ProcessingTimelineComponent = () => {
     const stageNumbers = stages.map((stage) => stage.stage);
 
     return pipelines
-      .map((p) => p.stages.map(({ stage }) => stage))
+      .map(({ stages: pStages }) => pStages.map(({ stage }) => stage))
```

Comment:
what does `p` stand up in `pStages` and why can't it just be `stages`?

Reply:
Unfortunately ESLint complains about that because it's shadowing the already defined `stages` argument. However, I have replaced it to the more verbose `pipelineStages` since I think it makes clearer what that is (and where the `p` comes from).